tests/phpunit/api/v4/Utils/ReflectionUtilsTest.php

```diff
@@ -97,15 +97,32 @@ public static function docBlockExamples() {
           'return' => ['nothing', 'something'],
         ],
       ],
+      'property_with_options' => [
+        'doc_block' => '/**
+         * Property Array
+         *
+         * @var array{
+         *    a: string,
+         *    z: int,
+         *    x: float|string|int,
+         *    }
+         */',
+        'parsed' => [
+          'description' => 'Property Array',
+          'type' => ['array'],
+          'comment' => NULL,
+          'keys' => ['a' => ['string'], 'z' => ['int'], 'x' => ['float', 'string', 'int']],
```

Comment:
Shouldn't this be 'shape' instead of 'keys'?

Reply:
@colemanw well - it's a matter of opinion - keys made sense to me - but I'm not wedded to it